core/spec.md

```diff
@@ -1941,6 +1946,32 @@ based on the authorization level of the client making the request.
 
 The following defines the specification-defined capabilities:
 
+#### `apis`
+- Name: `apis`
+- Type: Array of strings
+- Description: The list of APIs (beyond the APIs for the data model) that
+  are supported for read (`HTTP GET`) operations. This list is meant to allow
+  for clients/tooling to easily discover which of the APIs, that are not
+  related to the data model, are supported. Whether any of the APIs listed
+  below are supported for write operations can be discovered via the
+  `mutable` capability.
+- Note that it is allowable for the data that is available via more than one
+  mechanism to not be available via all mechanisms. For example, it is
+  possible for an implementation to support `GET /model` but not
+  `GET /?inline=model`.
+- Defined values:
+  - `/capabilities`
+  - `/export`
+  - `/model`
+- Values MUST start with `/`.
+- When not specified, the default value MUST be an empty list and no APIs
+  beyond ones for the data model are supported.
+- Implementations MAY define their own values but they MUST NOT conflict with
```

Comment:
It's worth thinking about what this means for spec evolution. If you *ever* add another API to the spec, that might mean that existing implementations could be broken if they already have an API with the same name.

Reply:
yup - same with extensions. It's a risk both sides take.